backend-rust/README.md

```diff
@@ -17,24 +17,83 @@ To run the services, the following dependencies are required to be available on
 
 ## Run the Indexer Service
 
-For instructions how to use the indexer run:
+For instructions on how to use the indexer run:
 
 ```
-ccdscan-indexer --help
+cargo run --bin ccdscan-indexer -- --help
 ```
 
+Example:
+
+```
+cargo run --bin ccdscan-indexer -- --node http://localhost:20001 --max-parallel-block-preprocessors 20 --max-processing-batch 20
```

Comment:
The example command for developers is convenient because it can be copy-pasted to get the service up and running fast without having to re-discover some reasonable settings for the service.

I added a comment to make people aware that they should avoid running the node against our public nodes, but it is more of a problem for our infrastructure if our public nodes' rate-limit (protection) does not work. So I think we should find out earlier rather than later if there is a problem and as such not completely discourage people from doing so.

I also want to highlight that for some developers (e.g. fixing a small bug in the indexer or working on a laptop), running a local node is just impractical and they will use our public nodes to some extend instead of going through the hassle of a local node.

Added comment:
```
Note: Since the indexer puts a lot of load, use your own local node whenever possible.
If using the public nodes, run the indexer as short as possible.
```

Reply:
I agree we should add  documentation describing how they work and the implication on resource usage (there is a TODO in the documentation already).